test/parallel_api/numeric/numeric.ops/in_place_scan.pass.cpp

```diff
@@ -19,64 +19,331 @@
 
 #include "support/test_config.h"
 #include "support/utils.h"
+#include "support/scan_serial_impl.h"
 
 #if TEST_DPCPP_BACKEND_PRESENT
-#    include "support/utils_sycl.h"
-#    include "support/sycl_alloc_utils.h"
-#    include "support/scan_serial_impl.h"
+#include <CL/sycl.hpp>
+#endif
+using namespace TestUtils;
+
+#if TEST_DPCPP_BACKEND_PRESENT
+static const char kMsgInclusiveScanNormal [] = "Wrong effect from inclusive scan (non-inplace)";
```

Comment:
As far as we introduce one class for each test case. I suggest incapsulating these string constant in a corresponding class  scope.

Reply:
Fixed.